### PR TITLE
Create CONDUCT.md

### PR DESCRIPTION
**Pull request by Victoriapm**
Initial doc, more detail about the specifics of the project should be added

### CODE REVIEW

File: CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Contributing To The Project
+
+If you decide to contribute anything, please do the following:
+
+1. Fork this repository
+1. Create a new branch from the latest `develop` (read more [here](https://guides.github.com/introduction/flow/))
```
Comment (sjackman) [inline CONDUCT.md:6]:
```suggestion
1. Create a new branch from the latest `master` (read more [here](https://guides.github.com/introduction/flow/))
```
We don't have a `develop` branch, but we do have a `master` branch.

File: CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+1. Start hacking on the new branch
+1. Commit and push to the new branch
+1. Make a pull request
```
Comment (sjackman) [inline CONDUCT.md:9]:
```suggestion
4. Open a pull request
```

File: CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Contributing To The Project
+
+If you decide to contribute anything, please do the following:
+
+1. Fork this repository
+1. Create a new branch from the latest `develop` (read more [here](https://guides.github.com/introduction/flow/))
+1. Start hacking on the new branch
+1. Commit and push to the new branch
```
Comment (sjackman) [inline CONDUCT.md:8]:
```suggestion
3. Commit and push to your new branch
```

File: CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Contributing To The Project
+
+If you decide to contribute anything, please do the following:
+
+1. Fork this repository
+1. Create a new branch from the latest `develop` (read more [here](https://guides.github.com/introduction/flow/))
+1. Start hacking on the new branch
```
Comment (sjackman) [inline CONDUCT.md:7]:
```suggestion
2. Start hacking on your new branch
```

**Review by sjackman — CHANGES_REQUESTED** [review]
A few minor suggestions. Otherwise looks great. Thank you for your contribution, Victoria!
Could you please rename this file from `CONDUCT.md` to `CONTRIBUTING.md`?

**Comment by sjackman** [comment]
@Victoriapm I'm just curious… how did you find this repo and this particular issue?
Are you involved with Hackseq 2019 in Vancouver 🇨🇦? https://www.hackseq.com

**Comment by sjackman** [comment]
Merged. Thank you, Victora, for your contribution to Talr! 🌟

File: CONDUCT.md (hunk 1 above)
Comment (Victoriapm) [reply CONDUCT.md:6]: Sure, good call!

**Comment by Victoriapm** [comment]
I found it on http://issuehub.io looking for repositories where I could contribute with some documentation for Hacktoberfest.

**Comment by sjackman** [comment]
Cool! Thanks for your help. Happy Hacktoberfest!

**Pull request merged**